apps/organization/src/organization.service.ts

```diff
@@ -623,26 +623,38 @@ export class OrganizationService {
         userId
       );
 
-      let orgIds;
-      if (0 > getOrgs?.organizations?.length) {
-        orgIds = getOrgs?.organizations?.map(item => item.id);
+      if (0 === getOrgs?.organizations?.length) {
+        throw new NotFoundException(ResponseMessages.organisation.error.organizationNotFound);
       }
-       
-      const orgEcosystemDetails = await this._getOrgEcosystems(orgIds);
-      if (!orgEcosystemDetails || !Array.isArray(orgEcosystemDetails) || 0 === orgEcosystemDetails.length) {
-        throw new NotFoundException(ResponseMessages.ecosystem.error.ecosystemDetailsNotFound);
+
+      let orgIds;
+      let updatedOrgs;
+
+      if ('true' === process.env.IS_ECOSYSTEM_ENABLE) {
+        if (0 > getOrgs?.organizations?.length) {
```

Comment:
Is this if condition required, since we already check the length above to throw error
`      if (0 === getOrgs?.organizations?.length) {`

Reply:
Yes, Correct. We have already checking the length, So there is no need to add `if (0 > getOrgs?.organizations?.length)`, this condition again.